http/handler_acp.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package http
+
+import (
+	"net/http"
+
+	"github.com/getkin/kin-openapi/openapi3"
+
+	"github.com/sourcenetwork/defradb/client"
+)
+
+type acpHandler struct{}
```

Comment:
suggestion: I'm not sure a whole new handler is justified here - suggest just defining `AddPolicy` on the store/db handler.